apps/src/templates/hintsDisplay.jsx

```diff
@@ -0,0 +1,57 @@
+module.exports = React.createClass({
```

Comment:
Looks like all the `/apps/src/templates/` `.jsx` files are uncapitalized. I think I'll leave this one as-is in this PR and open a second one to recapitalize all of em at once.